### ISSUE TITLE
Moderation

### ISSUE THREAD

**Issue by oestrich** [oestrich/gossip#30]
Moderation is hard. Let's start thinking about how to tackle this. [Raisin](https://github.com/oestrich/raisin) will be the tool that handles this.

**Comment by vadi2** [comment]
Has anything been decided on the code of conduct?

**Comment by oestrich** [comment]
Yeah, I checked with Zach over at the MUD Coders Guild and I'm going to use the same code that uses. I'll try to get that in place today.

https://github.com/mudcoders/code-of-conduct
